packages/top-app-bar/Title.tsx

```diff
@@ -0,0 +1,47 @@
+// The MIT License
+//
+// Copyright (c) 2019 Google, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import * as React from 'react';
+import * as classnames from 'classnames';
+import {cssClasses} from './constants';
+
+export interface TitleProps<T> extends React.HTMLProps<T> {
+  className?: string;
+  tag?: string;
+};
+
+const Title: <T extends {} = HTMLSpanElement>(props: TitleProps<T>) =>
```

Comment:
extend HTMLElement

Reply:
Even though the default tag is an HTMLSpanElement? 

see previous comment: https://github.com/material-components/material-components-web-react/pull/688/files/117d091ba8b8ef76b1082c339dbaae9175fb344e#r259917638

Reply:
Yes, you can do `<T extends HTMLElement = HTMLSpanElement>`.